src/App.tsx

```diff
@@ -19,6 +22,8 @@ const queryClient = new QueryClient()
 
 const App = withProviders(() => {
   useInitApp()
+  dayjs.extend(weekday)
+  dayjs.extend(localeData)
```

Comment:
plugins could be imported on demand (to reduce package size), as https://github.com/nervosnetwork/ckb-explorer-frontend/blob/develop/src/utils/date.ts#L9-L10